packages/react-core/src/demos/Card/Card.md

```diff
@@ -784,3 +787,257 @@ class CardViewBasic extends React.Component {
+      </DropdownItem>
+    ];
+    return (
+      <Card id="horizontal card" isExpanded={isCardExpanded}>
+        <CardHeader
+          onExpand={this.onCardExpand}
+          toggleButtonProps={{
+            id: 'toggle-button',
+            'aria-label': 'Actions',
+            'aria-labelledby': 'titleId toggle-button',
+            'aria-expanded': isCardExpanded
+          }}
+        >
+          <CardActions>
+            <Dropdown
+              onSelect={this.onActionSelect}
+              toggle={<KebabToggle onToggle={this.onActionToggle} />}
+              isOpen={isDropdownOpen}
+              isPlain
+              dropdownItems={dropdownItems}
+              position="right"
+            />
+          </CardActions>
+          {isCardExpanded && <CardTitle id="titleId">Getting Started</CardTitle>}
+          {!isCardExpanded && (
+            <Level hasGutter>
+              <CardTitle id="titleId">Getting Started</CardTitle>
+              <LabelGroup>
+                <Label icon={<InfoCircleIcon />} color="blue">
+                  Set up your cluster
+                </Label>
+                <Label icon={<InfoCircleIcon />} color="purple">
+                  Guided tours
+                </Label>
+                <Label icon={<InfoCircleIcon />} color="green">
+                  Quick starts
+                </Label>
+                <Label icon={<InfoCircleIcon />} color="orange">
+                  Learning resources
+                </Label>
+              </LabelGroup>
+            </Level>
+          )}
+        </CardHeader>
+        <CardExpandableContent>
+          <CardBody>
+            <Grid md={6} lg={3} hasGutter>
+              <Flex
+                spaceItems={{ default: 'spaceItemsLg' }}
+                alignItems={{ default: 'alignItemsFlexStart' }}
+                direction={{ default: 'column' }}
+              >
+                <Flex
+                  spaceItems={{ default: 'spaceItemsSm' }}
+                  alignItems={{ default: 'alignItemsFlexStart' }}
+                  direction={{ default: 'column' }}
+                  grow={{ default: 'grow' }}
+                >
+                  <Label icon={<InfoCircleIcon />} color="blue">
+                    Set up your cluster
+                  </Label>
+                  <p>Continue setting up your cluster to access all you cain in the Console</p>
+                  <List isPlain>
+                    <ListItem>
+                      <a href="#">Add identity provider</a>
+                    </ListItem>
+                    <ListItem>
+                      <a href="#">Configure alert receivers</a>
+                    </ListItem>
+                    <ListItem>
+                      <a href="#">Configure default ingress certificate</a>
+                    </ListItem>
+                  </List>
+                </Flex>
+                <Button component="a" variant="link" isInline icon={<ArrowRightIcon />} iconPosition="right">
```

Comment:
Should these have an `href`? I would probably expect them to since the other links in the columns have one (`href="#"`)

Reply:
I can add one